pkgs/by-name/tl/tlrc/package.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, installShellFiles
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "tlrc";
+  version = "1.7.1";
+
+  src = fetchFromGitHub {
+    owner = "tldr-pages";
+    repo = pname;
```

```suggestion
    repo = "tlrc";
```

Seems to be required to not use `pname` here. Not sure why but I got told so on previous PRs